infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -34,8 +30,10 @@
      * <p>
      * a cache of recent conflict keys and their latest global-log position.
      */
-    private final Cache<ConflictTxStream, Long> conflictCache;
-
+    private final Hashtable<ConflictTxStream, Long> cacheHashtable;
```

Comment:
Why Hashtable is used? Hashtable is not as performant as its counterpart. If synchronization is not needed, hashmap is recommended. If the map is used in a multi-threaded context, concurrenthashmap is more performant.